BUILD/familiars/drop.dat

```diff
@@ -42,7 +42,7 @@ Intergnat	item:BACON<150
 #
 # density 1.875 size 4 spleen consumables. boolean autoChooseFamiliar(location place) will already grab the amount needed for spleen
 # Here they are only used to grab extras.
-Grim Brother	prop:_grimFairyTaleDrops<5;!prop_boolean:_auto_thisLoopPlusNoncombat
```

Comment:
why remove this prop? grim brother gives +combat
without this prop it can be used when we are trying to maximize +noncombat

Reply:
wait... how come wiki has no mention of it? I got the notion that it has +5 combat chance from comments in autoscend. But I am not seeing any mention of this existing in wiki... weird